libnaboConfig.cmake.in

```diff
@@ -6,13 +6,7 @@
 # Compute paths
 get_filename_component(libnabo_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
 set(libnabo_INCLUDE_DIRS "@libnabo_include_dirs@")
- 
-# Our library dependencies (contains definitions for IMPORTED targets)
```

Comment:
For me it compiles with catkin (isolated, merge devel). But it is hard to say whether this holds for any catkin config :(. From my understanding of the catkin internals I don't see why it is needed. Cmake ```export(TARGET ...)``` is about exporting cmake targets.
Catkin should not require this by reading the libnaboConfig.cmake (in the devel space) for the variables and by inferring the build order based on the dependency encoded in the package.xml files.

How can we easily test more than just my setup? It is still not easy to let jenkins build pointmatcher with this PR for nabo, right? Same with ros build farm I suppose. I'll try to test it in a non catkin environment soon.